sdk/search/azure-search-documents/azure/search/documents/_queries.py

```diff
@@ -26,8 +26,12 @@ def filter(self, expression: str) -> None:
         self._request.filter = expression
 
     @property
-    def request(self):
-        """The service request for this operation."""
+    def request(self) -> Union[Type[AutocompleteRequest], Type[SearchRequest], Type[SuggestRequest]]:
+        """The service request for this operation.
+
+        :rtype: Union[Type[AutocompleteRequest], Type[SearchRequest], Type[SuggestRequest]]
```

Comment:
we should use "or" instead of Union in rtype. in the near future, pylint will start checking for this